src/projects/diff/hunk-ghost.html

```diff
@@ -0,0 +1,57 @@
+<dom-module id="hunk-ghost">
+  <style>
+    #container {
+      display: none;
+    }
+  </style>
+  <template>
+    <div id="container">
+      <svg version="1.0" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+      	 width="254px" height="137px" viewBox="0 0 254 137" enable-background="new 0 0 254 137" xml:space="preserve">
+      <g>
+      	<path fill="#E6E6E6" d="M254,129c0,4.418-3.582,8-8,8H11c-4.418,0-8-3.582-8-8V11c0-4.418,3.582-8,8-8h235c4.418,0,8,3.582,8,8V129
+      		z"/>
+      </g>
+      <g>
+      	<path fill="#FFFFFF" d="M251,126c0,4.418-3.582,8-8,8H8c-4.418,0-8-3.582-8-8V8c0-4.418,3.582-8,8-8h235c4.418,0,8,3.582,8,8V126z"
+      		/>
+      </g>
+      <path fill="#CCCCCC" d="M251,8c0-4.418-3.582-8-8-8H8C3.582,0,0,3.582,0,8v21h251V8z"/>
+      <rect x="30" y="49.997" opacity="0.6" fill="#C1E8C3" width="198.5" height="10.667"/>
+      <rect x="30" y="67" opacity="0.6" fill="#CCCCCC" width="198.5" height="10.667"/>
+      <rect x="30" y="85" opacity="0.6" fill="#D78585" width="87" height="10.667"/>
+      <rect x="30" y="103.667" opacity="0.6" fill="#C1E8C3" width="170" height="10.667"/>
+      <text transform="matrix(1 0 0 1 15.582 114.334)" opacity="0.6" fill="#C1E8C3" font-family="'Roboto Mono'" font-size="18">+</text>
+      <text transform="matrix(1 0 0 1 15.582 60.6636)" opacity="0.6" fill="#C1E8C3" font-family="'Roboto Mono'" font-size="18">+</text>
+      <text transform="matrix(1 0 0 1 15.582 95.667)" opacity="0.6" fill="#D78585" font-family="'Roboto Mono'" font-size="18">-</text>
+      <text transform="matrix(1 0 0 1 15.582 21.5)" opacity="1" fill="#FFFFFF" font-family="'Roboto Mono'" font-size="14" font-weight="600">[[_hunkText]]</text>
+      </svg>
+    </div>
+  </template>
+  <script>
+    Polymer({
+      is: 'hunk-ghost',
+      properties: {
+        hunkText: {
+          type: String,
+          value: "/dev/null"
+        },
+        _hunkText: {
+          type: String,
+          computed: '_trimText(hunkText)'
+        }
+      },
+      getSVG: function() {
+        return this.$.container.innerHTML;
+      },
+      _trimText: function(text) {
+        if (text.length > 26) {
+          // Take the last 23 characters and add '...' to the front.
+          // The maximum string length inside the ghost header is 26 chars.
```

Comment:
I have not checked, but I'm still working on getting this right.
There is this small problem where the image does not render the font specified in the svg code. Sometimes it uses the default browser font, sometimes a font that I don't recognize, but never the correct one.
When rendering the svg as an html element, however, the correct font does show up.